packages/shared/src/components/ShareBar.tsx

```diff
@@ -56,7 +56,7 @@ export default function ShareBar({ post }: { post: Post }): ReactElement {
             target="_blank"
             rel="noopener"
             onClick={() => onClick('whatsapp')}
-            icon={<WhatsappIcon />}
+            icon={<WhatsappIcon style={{ fontSize: '1.75rem' }} />}
```

Comment:
Why are we not using our pre-defined sizes?
```suggestion
            icon={<WhatsappIcon size="large" />}
```

Reply:
Already discussed with Chris - these icons are some old .svg and not components I created, so there is no size prop. We can create components in the same way I created them for other icons, but I don't think we need it, because it will be just a wrapper on a single .svg (no filled/outlined)

Reply:
Alrighty! Thanks for the notes!